src/main/java/abr/search_engine_abr/SearchUserAbr.java

```diff
@@ -1,6 +1,6 @@
 package abr.search_engine_abr;
 
-import ds.user_search_engine.SearchEngineFileGateway;
+import abr.user_avatar_image_management_abr.ds.user_search_engine.SearchEngineFileGateway;
```

Comment:
there would be package arrangement problems if this line runs no problem; if you are trying to import search engine gateways, import  SearchEngineDatabaseGateway instead, the filegateway is a class in ia & violates clean architecture